users/serializers.py

```diff
@@ -2,12 +2,23 @@
 from drf_extra_fields.fields import Base64ImageField
 from rest_framework import serializers
 
-from users.models import User
+from users.models import Followers, User
 
 
 class UserPublicSerializer(serializers.ModelSerializer):
     id = serializers.IntegerField(read_only=True)
     profile_photo = Base64ImageField()
+    relations = serializers.SerializerMethodField()
+
+    def get_relations(self, obj):
+        request = self.context.get("request")
+        if request:
```

Comment:
May it be null? Do we need this check?

Reply:
Testing doesn't work without this check

Reply:
The code shouldn't follow the tests. We need to add some fixture for dummy request in tests

Reply:
I see, by the way I've found what was the problem, so this check was removed and tests are ok